R/summarize_weighted_production.R

```diff
@@ -77,15 +137,63 @@ add_weighted_loan_production <- function(data, use_credit_limit = FALSE) {
   total_size_by_sector <- distinct_loans_by_sector %>%
     summarize(total_size = sum(.data[[loan_size]]))
 
-  data %>%
+  out <- data
+  metric_nm <- metric_col <- "production"
+  if (buildout) {
+    out <- add_buildout(out)
+    metric_nm <- "buildout"
+    metric_col <- "build_out"
```

Comment:
It would be nice to merge these two vectors into one; can we use either "buildout" or "build_out" everywhere?